stores/billa.js

```diff
@@ -5,8 +5,6 @@ const utils = require("./utils");
 const units = {
     kus: { unit: "ks", factor: 1 },
     kilogram: { unit: "kg", factor: 1 },
-    "p.dáv": { unit: "prací dávka", factor: 1 },
-    rl: { unit: "role", factor: 1 },
```

Comment:
Můžeš tohle mergnout s tím přechozím commitem (kde jsi to přidal)?

Reply:
Není jednodušší squash & merge tady nebo ev. v gitu ?

Reply:
Taky se dá, ale když to uděláš u sebe, tak to jednak vypadá lépe a více se naučíš :)